mathesar_ui/src/component-library/pagination/Pagination.svelte

```diff
@@ -112,39 +116,41 @@
               : `Goto Page ${_page}`}
             aria-selected={currentPage === _page}
             on:click={(e) => setPage(e, _page)}
+            on:mousedown={(e) => e.preventDefault()}
             data-tinro-ignore
           >
             {_page}
           </a>
         {:else}
-          <span
+          <button
             tabindex="0"
             role="link"
             aria-label={currentPage === _page
               ? `Current Page, Page ${currentPage}`
               : `Goto Page ${_page}`}
             class="page"
             on:click={(e) => setPage(e, _page)}
+            on:mousedown={(e) => e.preventDefault()}
             aria-selected={currentPage === _page}
           >
             {_page}
-          </span>
+          </button>
         {/if}
       </li>
     {/each}
 
     {#if pageInfo.end < pageCount}
       {#if pageInfo.end < pageCount - 1}
         <li>
-          <span
+          <button
             tabindex="0"
             role="link"
             aria-label="Goto Page {pageInfo.nextPageWindow}"
             on:click={(e) => setPage(e, pageInfo.nextPageWindow)}
           >
-            <Icon class="ellipsis" data={faEllipsisH} />
+            <Icon class="ellipsis" tabindex="-1" data={faEllipsisH} />
             <Icon class="arrow" data={faAngleDoubleRight} />
-          </span>
+          </button>
         </li>
       {/if}
       <li class:active={currentPage === pageInfo.end}>
```

Comment:
@pavish Is `class:active={currentPage === pageInfo.end}` necessary? Wouldn't this never be true since when `currentPage === pageInfo.end`, `pageInfo.end` will equal `pageCount`, and thus this block of code will not run?

Reply:
Yes, that's right. This can be removed.